homeassistant/components/rfxtrx/cover.py

```diff
@@ -98,17 +98,23 @@ def is_closed(self):
         """Return if the cover is closed."""
         return not self._state
 
-    def open_cover(self, **kwargs):
+    async def async_open_cover(self, **kwargs):
         """Move the cover up."""
-        self._send_command("roll_up")
+        await self._async_send(self._device.send_open)
+        self._state = True
+        self.async_write_ha_state()
 
```

Comment:
Can we leave out the `if self.hass:` here?

Reply:
yes, nothing should be calling services on the entity until it has been added to hass.

Reply:
Ok